configs/eslint-config-custom/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "eslint-config-custom",
+  "license": "MIT",
```

Comment:
Either remove the license field or point to the LICENSE.md file in the root of the repo